advanced/metrics.yaml

```diff
@@ -0,0 +1,187 @@
+components:
+  schemas:
+    AdvancedMetrics.Request:
+      type: object
+      properties:
+        choice:
+          type: string
+          enum: [event, latency, convergence]
+        event:
+          $ref: '#/components/schemas/AdvancedMetrics.Event.Request'
+        latency:
+          $ref: '#/components/schemas/AdvancedMetrics.Latency.Request'
+        convergence:
+          $ref: '#/components/schemas/AdvancedMetrics.Convergence.Request'
+
+    AdvancedMetrics.Event.Request:
```

Comment:
just want to understand the semantics of this.  I can see how metrics can be pulled and be consumed instantaneously (either as single shot or the consumer decides to shove that into a database after polling)

but does this pattern really apply to events?  events are usually fired and forget by the publisher.  Are we implying each traffic engine need to remember the last event triggered?  It doesn't feel like the right pattern for events.